app/views/layouts/application.html.erb

```diff
@@ -29,6 +29,7 @@
       <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
     <![endif]-->
 
+    <script src="https://maps.googleapis.com/maps/api/js?libraries=places&language=en&key=AIzaSyDWgc7p4WWFsO3y0MTe50vF4l4NUPcPuwE"></script>
```

Comment:
At some point soon, @david-days code will merge and he uses a `yield` command to enable subviews to add includes up top. 

https://github.com/david-days/plots2/blob/master/app/views/layouts/application.html.erb#L40

If his merge is near enough, we could just live with the inefficiency until then. I'll leave a note to that effect in #624.